### ISSUE TITLE
Does Model Training Use Half-Precision Training

### ISSUE THREAD

**Issue by mylwx** [bo-miao/SgMg#18]
Great work! I noticed that your training and inference code includes components related to mixed precision. For example, in the training code engine.py, there is with amp.autocast(enabled=args.amp), and in the inference code inference_ytvos.py, there is also with autocast(args.amp). However, in the shell scripts, the training script does not include the --amp parameter, while the inference script does. Could you clarify whether mixed precision training was used during the training process?

**Comment by bo-miao** [comment]
According to my memory, AMP is only used during training from scratch on Ref-YTV since it causes issues on Ref-COCOs with noisy labels, and AMP does not affect the model’s performance.
